source/help/getting-started/managing-members.rst

```diff
@@ -6,14 +6,35 @@ Managing Members
 Inviting Team Members
 ---------------------
 
-If enabled by your System Admin, you can add people to your team in one of three ways: You can send a direct invite; you can share a public team invite link; or if they already have an account on the server, you can add them to the team yourself.
+If you have the required permissions, you can add people to your team via a direct invite, a public team invite link, or if they already have an account on the server you can add them to the team yourself. 
+
+From version 5.16 your System Administrator can enable guest access to your server.  A guest is a special role that is restricted to only the channels they have been invited to. When this is enabled, you can invite guests or users external to the organization such as vendors or customers. 
 
 Direct Invites
 ~~~~~~~~~~~~~~
 
-Direct invites are invitation emails sent from your team's server directly to the invited person's email address. A link within the invitation directs them to an account creation page. Invitation links sent by email expire after 48 hours and can be used only one time. A System Admin can invalidate all active invitation links via **System Console > Security > Sign Up > Invalidate pending email invites** in prior versions or **System Console** > **Authentication** > **Signup** > **Invalidate pending email invites** in versions after 5.12.
+Direct invites are invitation emails sent from your team's server directly to the invited person's email address. A link within the invitation directs them to an account creation page. Invitation links sent by email expire after 48 hours and can be used only one time. A System Administrator can invalidate all active invitation links via **System Console > Authentication > Signup > Invalidate pending email invites** or, in versions prior to 5.12, **System Console > Security > Sign Up > Invalidate pending email invites**.
+
+**To send a direct invite**
```

Comment:
Some of these lists are not rendering as a list.

Reply:
Should all be fixed now.